src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -134,6 +134,16 @@ As a convenient shorthand, `transition:persist` can alternatively take a transit
 <video controls="" autoplay="" transition:persist="media-player">
 ```
 
+#### `transition:persist-props`
+<p><Since v="4.5.0" /></p>
+**Default**: `false`
+
+This allows you to control whether or not an island's props should be persisted upon navigation.
+
+By default, when you add `transition:persist` to an island, the state is retained upon navigation, but your component will re-render with new props. This is useful, for example, when a component receives page-specific props such as the current page's `title`.
+
+You can override this behavior with `transition:persist-props`. Adding this directive will keep an island's existing props (not re-render with new values) in addition to maintaining its existing state.
```

Comment:
``````suggestion
You can override this behavior with `transition:persist-props`. Adding this directive will keep an island's existing props (not re-render with new values) in addition to maintaining its existing state.

```astro
<ProductListing transition:persist-props=true products={products} />
```
``````
Thanks @matthewp ! Wording looks fine to me, but if they do have to set this to `true`, then we should probably show an example of that. (If they *don't* have to set it to true, and just merely adding it adds it as true, then I would choose to remove the "Default: false" from above.)

Reply:
They don't have to set it to true. It works like a boolean attribute where the presence of the attribute makes it be true. I can see how the "defaults to false" part might be confusing, even if its technically accurate, so we can remove if you think that's best.

Reply:
Great, then I'll remove the default line and we're good to go!